src/coreclr/vm/gcheaputilities.h

```diff
@@ -26,13 +26,20 @@ GVAL_DECL(gc_alloc_context, g_global_alloc_context);
 }
 #endif // !DACCESS_COMPILE
 
-// on MP systems, each thread has its own allocation chunk so we can avoid
-// lock prefixes and expensive MP cache snooping stuff
+struct GCThreadLocals
```

Comment:
Can we give it a name that is not GC specific? I would like to store `m_fPreemptiveGCDisabled`, `m_pFrame` and whatever else is used by stackwalking in it as well.

Naming ideas: `ThreadBuffer`, `RuntimeThread`, `RuntimeThreadLocals`, ...?

Rename `ThreadBuffer` in NativeAOT to whatever name we pick here. I would like those two structures to unify eventually.

Reply:
I'll name it `RuntimeThreadLocals` and update `ThreadBuffer` in NativeAOT.

Reply:
(make sure to update the cDAC impl too - https://github.com/dotnet/runtime/pull/103324)